frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -1,5 +1,186 @@
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.parentingPlan"
+          label="Parenting Plan"
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.other"
+          label="Other (describe)"
+        />
+        {/* {case.proofOfCompletedService_case && ( 
+          <TextArea label="Describe" dataKey="proofOfCompletedService_case.otherDocumentsServed" />
+        )} */}
+        <TextArea
+          dataKey="proofOfCompletedService_case.otherDocumentsServed"
+          label="Describe"
+        />
+      </Section>
+      <Section name="Complete section 2, or 3 AND 4"></Section>
+      <Section name="2. Service by Mail">
+        <p className="web-form-input">
+          Service by mail requires a signed receipt. You will need to attach the
+          receipt after printing the form.
+        </p>
+        <p className="web-form-input">
+          I served the following person by sending a copy of the documents
+          listed in section 3 by mail or commerical courier service to:
+        </p>
+        <TextInput
+          dataKey="proofofcompletedservice_person.fullname"
+          label="Name of Addressee"
+        />
+        <TextInput
+          dataKey="proofofcompletedservice_person.addressStreet"
+          label="Street Address"
+        />
+        <TextInput
+          dataKey="proofofcompletedservice_person.addressCity"
+          label="City"
+        />
+        <TextInput
+          dataKey="proofofcompletedservice_person.addressState"
+          label="State"
+        />
+        <TextInput
+          dataKey="proofofcompletedservice_person.addressZip"
+          label="Zip Code"
+        />
+        <p className="web-form-input">
+          I have attached a signed receipt proving delivery. It was signed by:
+        </p>
+        <Checkbox
+          dataKey="proofOfCompletedService_case.addressee"
+          label="the addressee personally"
+        />
+        <Checkbox
+          dataKey="proofOfCompletedService_case.someoneElse"
+          label="someone authorized by appointment or by law to receive service of process on behalf of the addressee"
+        />
+      </Section>
+      <Section name="3. Service by Third Person">
+        <Checkbox
+          dataKey="proofOfCompletedService_case.other"
+          label="I am over the age of 18, and"
+        />
+        <p className="web-form-input">
+          I am not a party or an attorney for a party to this action.
+        </p>
+        <p className="web-form-input">
+          I have not been convicted of a felony violation of a sex offsense{" "}
+          <small>(Listed in Utah Code 77-41-102(16)</small>.
+        </p>
+        <p className="web-form-input">
+          I am not a respondent in a protective order proceeding
+          <small>(Utah Code 78B-7-101 et seq.)</small>.
+        </p>
+      </Section>
+      <Section name="4. Service by Third Person (continued)"></Section>
```

Comment:
Are you planning on doing this section in a follow-up PR?

Reply:
Yes. In a new PR